src/lib/webpack/run-webpack.js

```diff
@@ -38,7 +38,12 @@ const devBuild = async (env, onprogress) => {
 				let serverAddr = `${protocol}://${host}:${chalk.bold(port)}`;
 				let localIpAddr = `${protocol}://${ip.address()}:${chalk.bold(port)}`;
 
-				process.stdout.write(chalk.green('\nCompiled successfully!!\n\n'));
+				if (stats.hasErrors()) {
+					process.stdout.write(chalk.red('\Build failed!\n\n'));
+				} else {
+					process.stdout.write(chalk.green('\nCompiled successfully!\n\n'));
+				}
```

Comment:
That could be possible but I don't know really. `webpack-dev-server` shows address always on first compilation.

We kinda don't inform users after they fixed the errors if anything occurred. Maybe the output should look like this:
```
You can view the application in browser.

   Local: ...
   On Your Network: ...

Build failed!
--Some error message--
Compiled successfully --date stamp--
Compiled successfully --date stamp--
Build failed!
--Some error message--
Compiled successfully --date stamp--
```

Let me know what you think (also @lukeed @developit @thangngoc89 )